packages/neos-ui-validators/src/StringLength/index.tsx

```diff
@@ -28,7 +30,12 @@ const StringLength = (value: any, validatorOptions: StringLengthOptions) => {
         return 'The minimum StringLength can not be less than zero';
     }
 
-    const stringLength = (value !== undefined && value !== null && value.toString) ? value.toString().length : 0;
+    let castedValue = value.toString ? value.toString() : '';
+    if (validatorOptions.ignoreHtml) {
+        const document = new DOMParser().parseFromString(castedValue, 'text/html');
```

Comment:
The code itself will work but we need to rename the variable because document is a reserved word. Like class or so.

Reply:
and isn't `castValue` grammatically the same as `castedValue` and the latter wrong (language-wise?) Never stop learning: https://grammarist.com/usage/casted/ – so, it's ok, a nice contrast to our modern TS codebase. ;)

Reply:
Haha, right, maybe we should call it `castleValue`? :)
Feel free to change both names.